mifospay/src/main/java/org/mifos/mobilewallet/mifospay/bank/fragment/DebitCardFragment.java

```diff
@@ -78,10 +78,14 @@ public boolean onEditorAction(TextView v, int actionId, KeyEvent event) {
     }
 
     public void okayClicked() {
-        showProgressDialog(Constants.PLEASE_WAIT);
+        if (mEtDebitCardNumber.getText().length() < 16) {
+            showToast("Please enter valid Debit Card Number");
```

Comment:
@shaliniv16 Can you please put the error message in String.xml file and call it from there